base_multi_image/views/image_view.xml

```diff
@@ -0,0 +1,150 @@
+                    </group>
+                    <group string="Name">
+                        <field name="name"/>
+                        <field name="storage"/>
+                        <field name="extension"/>
+                    </group>
+                </group>
+                <group>
+                    <group string="Options">
+                        <field
+                            name="url"
+                            attrs="{
+                                'invisible': [('storage', '!=', 'url')],
+                                'required': [('storage', '=', 'url')],
+                            }"
+                            widget="url"/>
+                        <field
+                            name="path"
+                            attrs="{
+                                'invisible': [('storage', '!=', 'file')],
+                                'required': [('storage', '=', 'file')],
+                            }"/>
+                        <field name="filename" invisible="1"/>
+                        <field
+                            name="file_db_store"
+                            attrs="{
+                                'invisible': [('storage', '!=', 'db')],
+                                'required': [('storage', '=', 'db')],
+                            }"
+                            filename="filename"/>
+                        <field
+                            name="attachment_id"
+                            attrs="{
+                                'invisible': [('storage', '!=', 'filestore')],
+                                'required': [('storage', '=', 'filestore')],
+                            }"/>
+                    </group>
+                    <group string="Preview">
+                        <field name="image_medium"
+                               widget="image"
+                               readonly="True"
+                               nolabel="1" />
+                    </group>
+                </group>
+                <group string="Comments">
+                    <field name="comments" nolabel="1" />
+                </group>
+            </sheet>
+        </form>
+    </field>
+</record>
+
+<record id="image_tree_view" model="ir.ui.view">
+    <field name="name">Multi image tree</field>
+    <field name="model">base_multi_image.image</field>
+    <field name="arch" type="xml">
+        <tree string="Images">
+            <field name="name"/>
+            <field name="storage"/>
+            <field name="owner_model"/>
+            <field name="owner_id"/>
+            <field name="sequence" invisible="True"/>
+        </tree>
+    </field>
+</record>
+
+<record id="image_kanban_view" model="ir.ui.view">
+    <field name="name">Product multi image kanban</field>
+    <field name="model">base_multi_image.image</field>
+    <field name="arch" type="xml">
+        <kanban string="Product Images">
+            <field name="name"/>
```

Comment:
https://github.com/OCA/server-tools/pull/1285#discussion_r194955489

Please add 'id' field as referenced by the above review in 11.0 migration.